src/infrastructure/HostNetworkManager_linux.go

```diff
@@ -594,6 +742,71 @@ func (h *HostNetworkManager) loadBridgeConfiguration(config domain.HostNetworkCo
 	return nil
 }
 
+func (h *HostNetworkManager) setConfigField(table string, value []domain.HostNetworkTrafficRule, config *domain.HostNetworkConfig) {
```

Comment:
I think this method should be renamed to be less generic. For example, setTrafficRulesConfig, and pass in the description of the parameters not value but rules.

Reply:
done